Front/useridsingleton.h

```diff
@@ -0,0 +1,32 @@
+#ifndef USERIDSINGLETON_H
+#define USERIDSINGLETON_H
+#include <iostream>
+#include <mutex>
+#include <thread>
+
+class UserIdSingleton
```

Comment:
If you want unique id for each user you don't need to create a whole class for that. You can add static mutex lock, static int counter and static function to get ID in user page for that

Reply:
Fixed